JDI.Light/JDI.Light/Utils/WinProcUtils.cs

```diff
@@ -1,65 +1,29 @@
 ﻿using System;
 using System.Diagnostics;
 using System.Linq;
-using System.Management;
 
 namespace JDI.Light.Utils
 {
     public static class WinProcUtils
     {
-        public static void KillAllRunWebDrivers()
-        {
-            try
-            {
-                foreach (var proc in Process.GetProcessesByName("chromedriver"))
-                    KillProcessTree(proc.Id);
-                foreach (var proc in from proc in Process.GetProcessesByName("firefox")
-                    let cmd = GetProcessCommandLine(proc.Id)
-                    where cmd.EndsWith("-foreground")
-                    select proc)
-                    proc.Kill();
-
-                foreach (var proc in Process.GetProcessesByName("IEDriverServer"))
-                    KillProcessTree(proc.Id);
-            }
-            catch (Exception)
-            {
-            }
-        }
+        public static DateTime? TestRunStartTime { get; set; }
 
-        private static string GetProcessCommandLine(int pid)
+        private static readonly string[] ProcessToKill =
         {
-            using (var searcher = new ManagementObjectSearcher("Select * From Win32_Process Where ProcessID=" + pid))
-            {
-                using (var moc = searcher.Get())
-                {
-                    foreach (var mo in moc)
-                        return mo["CommandLine"].ToString();
-                }
-            }
-            throw new ArgumentException("pid");
-        }
+            "chrome",
```

Comment:
won't it kill existing instances on Chorme itself?

Reply:
will kill it, yes. but created after the tests were started.